test/modules/TwinTester/DesiredPropertiesValidator.cs

```diff
@@ -27,6 +27,12 @@ public DesiredPropertiesValidator(RegistryManager registryManager, ModuleClient
             this.storage = storage;
             this.resultHandler = resultHandler;
             this.twinState = twinState;
+
+            moduleClient.SetConnectionStatusChangesHandler((status, reason) =>
+            {
+                Logger.LogInformation($"Applying failure threshold to desired property callback validation{Environment.NewLine}Detected change in connection status:{Environment.NewLine}Changed Status: {status} Reason: {reason}");
```

Comment:
Let me know if there is a better way to notate newlines

Reply:
FYI, https://stackoverflow.com/questions/1100260/multiline-string-literal-in-c-sharp.  I am ok with current syntax.

Reply:
Possible value of status are:
        Disconnected = 0,
        Connected = 1,
        Disconnected_Retrying = 2,
        Disabled = 3
I think we should only consider "Disconnected" to update LastTImeOfEdgeRestart, right?

Reply:
Good catch. Will change.

Reply:
I think that we should only consider Disconnected_Retrying because edgehub will come back before it enters the Disconnected state. Do you agree?